JPEGView/Config/KeyMap.txt

```diff
@@ -142,13 +144,15 @@ Ctrl+Shift+Plus         IDM_CONTRAST_CORRECTION_INC
 Ctrl+Alt+Plus           IDM_COLOR_CORRECTION_INC
 Ctrl+Plus               IDM_CONTRAST_INC
 Shift+Plus              IDM_GAMMA_INC
-Alt+Plus                IDM_SHARPEN_INC
+Alt+Plus				IDM_SATURATION_INC

```

Comment:
so this is a reassignment of the keys from sharpen to saturation right?

I would prefer making saturation (new feature) assign the new key rather than changing the older one

Reply:
That's right. 

Yea, I thought about just using a new shortcut, but thought it would be better to instead keep the shortcuts more consistent? Each one of the 3 modifier keys controls one of the 3 editing sliders on the left of the editing panel. I thought the sharpen slider was really of a different category (since it doesn't actually edit the photo, but instead is just a filter when viewing the image) and therefore should have a distinct type of shortcut. I thought that would be a more intuitive user experience? But I don't have strong feelings about this – I'll let you decide which way to go on this.